packages/key-management/test/restoreKeyAgent.test.ts

```diff
@@ -49,6 +77,14 @@ describe('KeyManagement/restoreKeyAgent', () => {
     // eslint-disable-next-line unicorn/consistent-function-scoping
     const getPassword: GetPassword = async () => Buffer.from('password');
 
+    it('assumes default stakeKeyDerivationPath if not present in serializable data', async () => {
+      const keyAgent = await restoreKeyAgent(inMemoryKeyAgentDataWithoutStakeDerivationPath, dependencies, getPassword);
+
+      for (const knownAddress of keyAgent.knownAddresses) {
```

Comment:
Won't catch any issue in this case, but it's a good practice to ensure the assertions are going to be called
```suggestion
       expect.assertions(1);
       for (const knownAddress of keyAgent.knownAddresses) {
```

Reply:
This should be outside of `for` loop. Typically `expect.assertions` is added as 1st line of the test